operator/controllers/lifecycle/keptnevaluation/controller_test.go

```diff
@@ -35,16 +35,16 @@ func TestKeptnEvaluationReconciler_fetchDefinitionAndProvider(t *testing.T) {
 		{
 			name: "keptn metrics",
 			namespacedDefinition: types.NamespacedName{
-				Namespace: providers.KLTNamespace,
```

Comment:
I would still use a var to define this so we can change it in a single place. Maybe let's add it to the beginning of this test